project/dexter-executor/src/java/com/samsung/sec/dexter/executor/cli/AnalyzedFileInfo.java

```diff
@@ -9,7 +9,7 @@
 	private long lastModifiedTime;
 	private String filePathString;
 	
-	private static final long TIME_OFFSET = 50; // msec 
+	private static final long TIME_OFFSET = 200; // msec 
```

Comment:
It is not file monitoring time-offset. It's to fix internal error which home monitor reports the change event twice for one chane. and This error occurred when user uses 'Ultra-edit' editor.